api_app/requirements.txt

```diff
@@ -1,16 +1,17 @@
 # API
 aiohttp==3.7.4.post0
```

Comment:
This needs to change back to 3.7.3 or 3.7.4 - dependabot is complaining -- note this also exists in the resource processor reqs